src/libraries/System.Text.Json/tests/Serialization/Stream.ReadTests.cs

```diff
@@ -12,9 +12,10 @@ namespace System.Text.Json.Serialization.Tests
     public static partial class StreamTests
     {
         [Fact]
-        public static async Task NullArgumentFail()
+        public static async Task ReadNullArgumentFail()
         {
             await Assert.ThrowsAsync<ArgumentNullException>(async () => await JsonSerializer.DeserializeAsync<string>((Stream)null));
+            await Assert.ThrowsAsync<ArgumentNullException>(async () => await JsonSerializer.DeserializeAsync((Stream)null, (Type)null));
```

Comment:
While we are doing this, let's add all permutations. For example: have the stream parameter be null, but the type parameter be non-null.

Reply:
fixed in 7a6e0cbc